### DISCUSSION TITLE
I want to know my primary email adress to reset password and sign in.

### DISCUSSION THREAD

**Discussion by toxic-devil** [community/community#31007] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know my email adress which I have sign in(ed) github. I have several accounts in github so I don't remember the password and email adress of one of my account so I wanna reset my password. Initially I wanna know my email adress to reset my password so please help me to find my email adress..

**Comment by jfullstackdev** [comment]
go to your Settings, 

or simply click this when you are logged in

https://github.com/settings/emails

**Reply by toxic-devil** [reply]
> go to your Settings,
> 
> or simply click this when you are logged in
> 
> https://github.com/settings/emails

I am not signed in that account

**Reply by Kirtishukla2004** [reply]
hey ,

 github  is a social coding platform used by software developers and companies all around the world. GitHub profiles generally include the individual's name, company, location and sometimes the profile owner's email address.



> Step 1: Locate a non-forked repository
Step 2: Find a commit by the user
Step 3: Convert to the patch view to locate the email address

**Reply by Preetiraj3697** [reply]
Go to Google and search it bro.

**Reply by DTDID** [reply]
> Go to Google and search it bro.

Super constructive. Thanks for being so helpful.

**Comment by Kirtishukla2004** [comment, marked as answer]
if you are still not able to find go the the link below for better understanding
https://www.thereach.io/blog/articles/how-to-find-github-user-email-addresses

**Reply by DTDID** [reply]
:( the blog is no longer available. Any idea where I can find the same content/article?

**Reply by Kirtishukla2004** [reply]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email 
check this official github site i  thik this will be helpful
